src/components/OrdersWidgetDemo/index.tsx

```diff
@@ -1,39 +1,37 @@
 import React from 'react'
-import Tabs, { TabItemType, TabThemeType } from 'components/common/Tabs/Tabs'
+import Tabs, { getTabTheme, TabItemInterface } from 'components/common/Tabs/Tabs'
 import { OrdersWidgetDemo as Wrapper } from './OrdersWidgetDemo.styled'
 import { ActiveOrdersContent } from './ActiveOrdersContent'
 
-const tabItems: TabItemType[] = [
+const tabItems: TabItemInterface[] = [
   {
     id: 1,
-    title: 'Active Orders',
+    tab: 'Active Orders: ' + 5,
```

Comment:
nit: why tab is a better name than title? I like it more with title. Agree on removing `count`.

Reply:
because before `title: string` now `tab: ReactNode` which is a better vernacular imo